src/index.html

```diff
@@ -15,8 +16,72 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__left">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </div>
+
+      <div class="header__right">
+        <article class="product">
+          <h2 class="product__title product__elect">APPLE</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">SAMSUNG</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">SMARTHONES</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title product__elect">LAPTOPS & COMPUTERS</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">GADGETS</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">TABLETS</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">PHOTO</h2>
+        </article>
+        <article class="product">
+          <h2 class="product__title">VIDEO</h2>
+        </article>
```

Comment:
Its a navigation so use appropriate tag `<nav>` for it. Ans use `<ul>` for list and `<li>` for list item. Each list item should be a link (`<a>`) with correct href attribute